content/en/docs/duplicate-certificate-limit.md

```diff
@@ -23,6 +24,31 @@ set" of hostnames for your certificate would be `[example.com]`. This rate limit
 is exceeded when a subscriber requests a certificate for the same "exact set" of
 hostnames more than 5 times in a single week.
 
+## Wait for Reset
```

Comment:
The majority of the section headings begin with a gerund verb. I suggest we do the same here as it's a user action.